### ISSUE TITLE
Quicklinks contrast inadequate

### ISSUE THREAD

**Issue by jvogtuw** [uw-drupal/uw_boundless#1]
The brown text on tan background on the Quicklinks button in the header does not meet accessibility contrast standards.
